rudder-lang/src/technique.rs

```diff
@@ -64,33 +66,35 @@ struct MethodCall {
     component: String,
 }
 
-pub fn translate_file(json_file: &Path, rl_file: &Path) -> Result<()> {
-    let config_data = fs::read_to_string("data/config.toml").expect("Cannot read config.toml file");
-    let config: toml::Value = toml::from_str(&config_data).expect("Invalig config.toml file");
+pub fn translate_file(json_file: &Path, rl_file: &Path) -> Result<()>
+{
+    let input_filename = &json_file.to_string_lossy();
+    let output_filename = &rl_file.to_string_lossy();
+    let config_filename = "data/config.toml";
+    let file_error = |filename: &str, err| err!(Token::new(&filename.to_owned(), ""), "{}", err);
```

Comment:
I'm not sure a file error should be represented using a Token, but this will do the job for now.

Reply:
Indeed, Clunky... but it allows to have only 1 format function to deal with 100% of errors. 
Tokens store both a filename and it's location